src/renderer/features/context-menu/menus/song-context-menu.tsx

```diff
@@ -28,6 +29,7 @@ export const SongContextMenu = ({ items, type }: SongContextMenuProps) => {
             bottomStickyContent={<ContextMenuPreview items={items} itemType={type} />}
         >
             <PlayAction ids={ids} itemType={LibraryItem.SONG} songs={items} />
+            <PlayTrackRadioAction song={items[0]} />
```

Comment:
It would also make sense to add this to the QueueContextMenu and PlaylistSongContext menus as those are both derivatives of the `Song` type.

Reply:
@jeffvli sure but QeueContextMenu and PlaylistSongContext are for multiple ids? How would i choose the one to pass to PlayTrackRadioAction?

Reply:
The same way it's being handled in the SongContextMenu? Either pick the first item, or disable the item if more than one is selected. Preferably disable now that I think about it.

Reply:
@jeffvli done!